pkg/git/git.go

```diff
@@ -175,6 +175,11 @@ type Repository interface {
 	Tags() (storer.ReferenceIter, error)
 }
 
+type CleanableRepository interface {
```

Comment:
Is this interface used somewhere?

Reply:
It wasn't in the end. The other functions just return the `*Repo` struct pointer and not the interface, and I didn't want to break with that style for the new function (by returning a `CleanableRepository` instead of a `*Repo`).

I can remove it if you'd like.

Reply:
Yes please. :)